gui/__tests__/containers/SignIn.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import createStore from 'connectors/redux'
+import syncFetch from 'test-helpers/syncFetch'
+import { mount } from 'enzyme'
+import { Provider } from 'react-redux'
+import { Switch, Route } from 'react-static'
+import { MemoryRouter } from 'react-router'
+import SignIn from 'containers/SignIn'
+
+const RedirectApp = () => (
+  <div>Behind authentication</div>
+)
+const mountSignIn = (store, props) => (
+  mount(
+    <Provider store={store}>
+      <MemoryRouter initialEntries={['/signin']}>
+        <Switch>
+          <Route exact path='/signin' component={SignIn} />
+          <Route exact path='/' component={RedirectApp} />
+        </Switch>
+      </MemoryRouter>
+    </Provider>
+  )
+)
+
+const submitForm = (wrapper) => {
+  wrapper.find('input#email').simulate('change', { target: { value: 'some@email.net' } })
```

Comment:
It may be worthwhile bringing in the `fill-in` helper from the link faucet project https://github.com/smartcontractkit/link_faucet/blob/master/spec/javascript/support/fill-in.js